src/veins/modules/analogueModel/SimpleObstacleShadowing.h

```diff
@@ -42,6 +42,30 @@ class Signal;
  */
 class SimpleObstacleShadowing : public AnalogueModel {
 protected:
+    struct CacheKey {
```

Comment:
This can possibly be simplified to just use a `std::pair<Coord, Coord>`. In this case, one would just have to provide an comparator to the `CacheEntries`. Similarly, we could use a `std::unordered_map` instead of `std::map`.

None of this is really necessary though, since this PR moves existing code which has sufficed for now.